integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -70,18 +70,14 @@ master:
     embedded: 19200
     rpc: 19998
     web: 19999
-  # You can specify hostNetwork and dnsPolicy here to override the default
-  # hostNetwork: false
+  hostNetwork: false
+  # dnsPolicy will be ClusterFirstWithHostNet if hostNetwork: true
+  # and ClusterFirst if hostNetwork: false
+  # You can specify dnsPolicy here to override this inference
```

Comment:
Per https://github.com/Alluxio/alluxio/issues/11526#issuecomment-641687729, inferring dnsPolicy from hostNetwork, but not inferring hostNetwork from whether it's root.